### DISCUSSION TITLE
`CocAction` and `CocFix` were removed?

### DISCUSSION THREAD

**Discussion by amgodoi** [neoclide/coc.nvim#3878] [Q&A] [answered]
Why `CocAction` and `CocFix` were removed?

Is there any way to substitute them?

I just added them to my .vimrc:
command! -nargs=* -range CocAction :call coc#rpc#notify('codeActionRange', [<line1>, <line2>, <f-args>])
command! -nargs=* -range CocFix    :call coc#rpc#notify('codeActionRange', [<line1>, <line2>, 'quickfix'])

Is it wrong?

**Comment by chemzqm** [comment, marked as answer]
Those command names confuse some users, you can create range command yourself, checkout `:h CocAction('codeActionRange')`
